CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ This section will walk you through how to get started working with the code.
 
 Verify that node.js is [installed](https://nodejs.org/en/download/).
 
-Verify that Yarn v1 is [installed](https://classic.yarnpkg.com/en/docs/install).
+Verify that pNPM is [installed](https://pnpm.io/installation).
```

Comment:
A typo here. **P**NPM

Reply:
I swear I saw this somewhere on their website, but I can't find it anymore. I'll update.